modules/pymol/plugins/installation.py

```diff
@@ -55,8 +53,7 @@ def cmp_version(v1, v2):
     if v2 == '':
         return 1
     try:
-        from distutils.version import StrictVersion as Version
-        return cmp(Version(v1), Version(v2))
+        return int(v1.split(".") < v2.split("."))
```

Comment:
Fails assertions:
```python
assert cmp_version("3.1.0", "3.2.0") == -1
assert cmp_version("3.2.0", "3.2.0") == 0
assert cmp_version("3.5.0", "3.2.0") == 1
assert cmp_version("3.10.0", "3.2.0") == 1
```

I believe for a couple of reasons: the original `cmp` function returns -1, 0, 1 rather than 0/1. There's also a lex comparison before the integer cast, so "3.10.0" will be seen to be less than "3.2.0". On my local branch that fixes this I have:

```
        v1_parts = list(map(int, v1.split('.')))
        v2_parts = list(map(int, v2.split('.')))
        return (v1_parts > v2_parts) - (v1_parts < v2_parts)
```
which yields a list of versions expressed as a list of `int`s and compares them

Reply:
oops, my bad thanks for catching this!